KavtorinDD/lab_1

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+int main()
+{
+
+	double num1, num2;
+	char sign, ch;
+	do
+	{
+		cout << "Enter num1:";
+		cin >> num1;
+		cout << "Enter operation:";
+		cin >> sign;
+		cout << "Enter num2:";
+		cin >> num2;
+
+		switch (sign)
+		{
+		case '+': cout << "Results:" << num1 + num2 << "If you finished, press ESC" << endl;
+			break;
+		case '-': cout << "Results:" << num1 - num2 << "If you finished, press ESC" << endl;
+			break;
+		case '*': cout << "Results:" << num1 * num2 << "If you finished, press ESC" << endl;
```

Comment:
extract case '?': `cout << "Results:" << num1 ? num2 << "If you finished, press ESC" << endl;` from switch..case in order to reduce code duplication